### PR TITLE
Apply the same requirement rule to symfony/event-dispatcher and other symfony components

### PR DESCRIPTION
**Pull request by kblais**
Avoid having to add the new version when a new minor is released (for example 4.3 is out), and standardize the requirements to Symfony components

### CODE REVIEW

**Comment by brexis** [comment]
@kblais Version 4.3 of Symfony Workflow introduces a nullable transition on events, witch triggers a "Call to a member function getTos() on null". Do you have any idea on how to fix that? [See travis](https://travis-ci.org/brexis/laravel-workflow/builds/574759367?utm_source=github_status&utm_medium=notification)

**Comment by kblais** [comment]
Yeah, I saw that after pushing... Gonna work on that later today, and complete the PR

File: src/WorkflowRegistry.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function __construct(array $config)
         $this->config = $config;
         $this->dispatcher = new EventDispatcher();
 
-        $subscriber = new WorkflowSubscriber();
+        $subscriber = new WorkflowSubscriber($this->dispatcher);
```
Comment (brexis) [inline src/WorkflowRegistry.php]: Is this line necessary?

Comment (kblais) [reply src/WorkflowRegistry.php]: Nope, my bad, removing it

**Comment by brexis** [comment]
One more thing, we need at least to fire the EnteredEvent event if the transition is null like Symfony Workflow does.

**Pull request merged**